lib/Doctrine/DBAL/Schema/OracleSchemaManager.php

```diff
@@ -137,7 +139,7 @@ protected function _getPortableTableColumnDefinition($tableColumn)
         }
 
         // Default values returned from database sometimes have trailing spaces.
-        $tableColumn['data_default'] = trim($tableColumn['data_default']);
+        $tableColumn['data_default'] = trim((string) $tableColumn['data_default']);
```

Comment:
Looks like this is redundant. The value is checked against null and trimmed again a few lines later.

Reply:
It's trimmed with other characters than whitespace. I'm changing this to be guarded with `!== null` condition instead`.